lib/ember_cli/route_helpers.rb

```diff
@@ -14,7 +16,8 @@ def mount_ember_app(app_name, to:, **options)
           format: :html,
         )
 
-        scope constraints: ::EmberCli::HtmlConstraint.new do
+        scope constraints: ::EmberCli::MultiConstraint.new(::EmberCli::HtmlConstraint.new,
```

Comment:
I'm not sure what to do about this one as I can't find an aesthetically pleasing way to format this one. Suggestions?